TestRktStop on stage1-kvm often fails on SemaphoreCI

Recently we are seeing an increasing number of failure on SemaphoreCI, while testing stage1-kvm. They are failing on `TestRktStop`, as follows:

```
=== RUN   TestRktStop
--- FAIL: TestRktStop (21.87s)
    rkt_tests.go:115: Running command: /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 --insecure-options=image prepare /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/test-tmp/rkt-stop-test.aci
    rkt_tests.go:115: Running command: /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 --insecure-options=image run-prepared --interactive b22e5d1a-2a14-44d5-add6-986757de1fbb
    rkt_stop_test.go:92: Running test #0, /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 stop b22e5d1a-2a14-44d5-add6-986757de1fbb
    rkt_tests.go:115: Running command: /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 stop b22e5d1a-2a14-44d5-add6-986757de1fbb
    rkt_tests.go:115: Running command: /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 --insecure-options=image prepare /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/test-tmp/rkt-stop-test.aci
    rkt_tests.go:115: Running command: /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 --insecure-options=image run-prepared --interactive ad47a77a-f0e7-4442-9bd1-f42afb50c009
    rkt_stop_test.go:92: Running test #1, /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 stop --force ad47a77a-f0e7-4442-9bd1-f42afb50c009
    rkt_tests.go:115: Running command: /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 stop --force ad47a77a-f0e7-4442-9bd1-f42afb50c009
    rkt_tests.go:115: Running command: /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 --insecure-options=image prepare /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/test-tmp/rkt-stop-test.aci
    rkt_tests.go:115: Running command: /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 --insecure-options=image run-prepared --interactive 0e3a3cb9-28c2-4832-872f-b321033252d6
    rkt_stop_test.go:92: Running test #2, /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 stop --uuid-file=/tmp/rkt-TestRktStop-320355424/uuid-file630404159
    rkt_tests.go:115: Running command: /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/tmp/functional/rkt --dir=/tmp/datadir-393981548 --local-config=/tmp/localdir-097164507 --system-config=/tmp/systemdir-476420222 --user-config=/tmp/userdir-881331141 stop --uuid-file=/tmp/rkt-TestRktStop-320355424/uuid-file630404159
    rkt_stop_test.go:107: Expected pod "0e3a3cb9-28c2-4832-872f-b321033252d6" to be exited, but it is "running"
panic: Failed to remove temporary data directory "/tmp/datadir-393981548": remove /tmp/datadir-393981548/pods/run/0e3a3cb9-28c2-4832-872f-b321033252d6/stage1/rootfs/sys: device or resource busy [recovered]
    panic: Failed to remove temporary data directory "/tmp/datadir-393981548": remove /tmp/datadir-393981548/pods/run/0e3a3cb9-28c2-4832-872f-b321033252d6/stage1/rootfs/sys: device or resource busy

goroutine 418 [running]:
testing.tRunner.func1(0xc820396fc0)
    /usr/local/go/src/testing/testing.go:450 +0x171
github.com/coreos/rkt/tests/testutils.(*dirDesc).cleanup(0xc82020d640)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/tests/testutils/ctx.go:67 +0x27c
github.com/coreos/rkt/tests/testutils.(*RktRunCtx).Cleanup(0xc82020d740)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/tests/testutils/ctx.go:179 +0x1be
runtime.Goexit()
    /usr/local/go/src/runtime/panic.go:320 +0xfb
testing.(*common).FailNow(0xc820396fc0)
    /usr/local/go/src/testing/testing.go:336 +0x36
testing.(*common).Fatalf(0xc820396fc0, 0xc9e360, 0x2a, 0xc8203afe18, 0x2, 0x2)
    /usr/local/go/src/testing/testing.go:379 +0x83
github.com/coreos/rkt/tests.TestRktStop(0xc820396fc0)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/tests/rkt_stop_test.go:107 +0x1254
testing.tRunner(0xc820396fc0, 0x11734c8)
    /usr/local/go/src/testing/testing.go:456 +0x98
created by testing.RunTests
    /usr/local/go/src/testing/testing.go:561 +0x86d

goroutine 1 [chan receive]:
testing.RunTests(0xd1e130, 0x1172c40, 0x60, 0x60, 0xbcfd01)
    /usr/local/go/src/testing/testing.go:562 +0x8ad
testing.(*M).Run(0xc82003df08, 0xc820164686)
    /usr/local/go/src/testing/testing.go:494 +0x70
main.main()
    github.com/coreos/rkt/tests/_test/_testmain.go:276 +0x116

goroutine 17 [syscall, 31 minutes, locked to thread]:
runtime.goexit()
    /usr/local/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 156 [chan receive, 15 minutes]:
github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels.func2(0xc82002c860, 0xc8203bbe60, 0xc82002c868)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:177 +0x5e
created by github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:190 +0x127

goroutine 158 [chan receive, 15 minutes]:
github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels.func2(0xc82002c8d8, 0xc82057ec00, 0xc82002c8e0)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:177 +0x5e
created by github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:190 +0x127

goroutine 138 [chan receive, 15 minutes]:
github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels.func2(0xc820112188, 0xc820132db0, 0xc820112190)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:177 +0x5e
created by github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:190 +0x127

goroutine 170 [chan receive, 15 minutes]:
github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels.func2(0xc820106490, 0xc820550720, 0xc820106498)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:177 +0x5e
created by github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:190 +0x127

goroutine 168 [chan receive, 15 minutes]:
github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels.func2(0xc8201063e8, 0xc82038f6e0, 0xc8201063f0)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:177 +0x5e
created by github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:190 +0x127

goroutine 166 [chan receive, 15 minutes]:
github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels.func2(0xc820106360, 0xc82038e7b0, 0xc820106368)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:177 +0x5e
created by github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:190 +0x127

goroutine 160 [chan receive, 15 minutes]:
github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels.func2(0xc82002c960, 0xc82057fc50, 0xc82002c968)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:177 +0x5e
created by github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:190 +0x127

goroutine 140 [chan receive, 15 minutes]:
github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels.func2(0xc820112238, 0xc820133e30, 0xc820112240)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:177 +0x5e
created by github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).AsyncInteractChannels
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:190 +0x127

goroutine 354 [chan send]:
github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).expectTimeoutRegexFind.func2(0xdf8475800, 0xc8201109f0, 0xc788e0, 0x21, 0xc820224ed0, 0xc820015500)
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:239 +0x387
created by github.com/coreos/rkt/vendor/github.com/coreos/gexpect.(*ExpectSubprocess).expectTimeoutRegexFind
    /home/runner/rkt/builds/build-rkt-kvm/build-rkt-1.13.0+git/gopath/src/github.com/coreos/rkt/vendor/github.com/coreos/gexpect/gexpect.go:240 +0x17c
exit status 2
FAIL    github.com/coreos/rkt/tests 1897.924s
```

One such example is https://semaphoreci.com/coreos/rkt/branches/pull-request-3090/builds/1

--- a/tests/rkt_stop_test.go
+++ b/tests/rkt_stop_test.go
@@ -90,7 +90,9 @@ func TestRktStop(t *testing.T) {
 
 		runCmd := fmt.Sprintf("%s %s %s", ctx.Cmd(), tt.cmd, args)
 		t.Logf("Running test #%d, %s", i, runCmd)
-		spawnOrFail(t, runCmd)
+		exitStatus := 0
+		// Issue stop command, and wait for it to complete
+		spawnAndWaitOrFail(t, runCmd, exitStatus)
 
 		// Make sure the pod is stopped
 		var podInfo *podInfo
@@ -107,7 +109,7 @@ func TestRktStop(t *testing.T) {
 			t.Fatalf("Expected pod %q to be exited, but it is %q", podUUID, podInfo.state)
 		}
 
-		exitStatus := 0
+		exitStatus = 0
 		if tt.expectKill {
 			exitStatus = -1
 		}

